tests/headers/struct_with_incomplete_array.hpp

```diff
@@ -0,0 +1,3 @@
+struct bpf_raw_tracepoint_args {                                                                                              
+        unsigned long long args[0];                                                                                                        
```

Comment:
Could you add a test with another struct with a smaller integer type (like `short`, maybe?).

Reply:
Hmm, good catch. The alignment is still 8 with the current code when I change to a smaller integer type.

Reply:
Oh, I see, for some reason I thought the should always have pointer alignment... I think [T; 0] is the right thing to do, these structs aren't safe to copy generally anyway.